src/main/resources/com/mabl/integration/jenkins/MablStepBuilder/help-proxyCredentials.html

```diff
@@ -0,0 +1,3 @@
+<div>
```

Comment:
Do we still need these help HTML refs, if we're using the internal Jenkins proxy configs?

Reply:
Yes as I added a call to the health/live endpoint as part of the form validation (which was not there before).